EU4toV2/Source/EU4World/Leader/EU4Leader.h

```diff
@@ -32,7 +34,6 @@ class Leader: commonItems::parser
 	int shock = 0;
 	int maneuver = 0;
 	int siege = 0;
-	std::string country;
 	int leaderID = 0;
```

Comment:
Since this has the potential to be unset, maybe use an optional<int>

Reply:
actually I'd rather let it default to 0 than work with id-less leaders. Reverted optional to 0.